src/components/EnhancedDataTable/EnhancedDataTable.spec.tsx

```diff
@@ -707,4 +707,52 @@ describe('<EnhancedDataTable />', () => {
     userEvent.click(getByTestId('enhancedDataTable-filterBar-actions-0'));
     expect(handler).toHaveBeenCalledTimes(1);
   });
+
+  it('should render addtional row actions', async () => {
+    const handlerA = jest.fn();
+    const handlerB = jest.fn();
+    const actions = [
+      { icon: GetApp, handler: handlerA },
+      { icon: Edit, handler: handlerB },
+    ];
+    const { getAllByTestId, getByTestId } = render(
+      <EnhancedDataTable
+        columns={columns}
+        fetchData={fetchDataFn}
+        rowActions={actions}
+      />
+    );
+    await waitFor(() => {});
+    expect(
+      getAllByTestId('enhancedDataTable-body-row-action-0')[0]
```

Comment:
können wir hier vielleicht den Row-Index verwenden bzw. diesen direkt in der test id verwenden, anstatt `getAllByTestId` aufzurufen und dann das Item aus dem erhaltenen Array auszuwählen? 
Die jetzige Lösung ist natürlich nicht "falsch", aber es wäre dann einheitlich mit den anderen Test-Ids und würde die Verwendung dieser IDs erleichtern (Werden zum Teil ja auch in anderen Projekten und ggf. irgendwann sogar in E2E Tests verwendet)

```suggestion
      getByTestId('enhancedDataTable-body-row-0-action-0')
```